 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:27:26] "GET /accept HTTP/1.1" 200 -
python: can't open file 'webservicrver.py': [Errno 2] No such file or directory
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:31:38] "GET /accept?project=test&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:31:47] "GET /accept?project=test&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/SQLiteHandler.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:33:25] "GET /accept?project=test&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:33:49] "GET /accept?project=test&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:34:36] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
  File "webservice.py", line 38
    "projecttable", "*", "ProjectID=" + projectid))
                                                  ^
SyntaxError: invalid syntax
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:35:50] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py", line 38, in acceptRequest
    "projecttable", "*", "ProjectID=" + str(projectid))
NameError: global name 'projectid' is not defined
24.55.154.193 - - [18/Mar/2016 02:35:50] "GET /accept?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:35:50] "GET /accept?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:35:50] "GET /accept?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:35:51] "GET /accept?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:35:51] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:35:51] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:36:05] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:36:17] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py", line 40, in acceptRequest
    return str(currentMemberList[1])
IndexError: list index out of range
24.55.154.193 - - [18/Mar/2016 02:36:17] "GET /accept?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:36:17] "GET /accept?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:36:17] "GET /accept?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:36:17] "GET /accept?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:36:17] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:36:17] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:36:25] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:37:10] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py", line 39, in acceptRequest
    memberListasList = currentMemberList.split(",");
AttributeError: 'list' object has no attribute 'split'
24.55.154.193 - - [18/Mar/2016 02:37:10] "GET /accept?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:37:10] "GET /accept?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:37:10] "GET /accept?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:37:10] "GET /accept?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:37:10] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:37:10] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:38:08] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py", line 39, in acceptRequest
    memberListasList = currentMemberList.split();
AttributeError: 'list' object has no attribute 'split'
24.55.154.193 - - [18/Mar/2016 02:38:09] "GET /accept?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:38:09] "GET /accept?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:38:09] "GET /accept?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:38:09] "GET /accept?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:38:09] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:38:09] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:38:20] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py", line 39, in acceptRequest
    memberListasList = currentMemberList[0].split();
AttributeError: 'tuple' object has no attribute 'split'
24.55.154.193 - - [18/Mar/2016 02:38:20] "GET /accept?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:38:21] "GET /accept?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:38:21] "GET /accept?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:38:21] "GET /accept?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:38:21] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:38:21] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:38:35] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:38:52] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:39:01] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:39:06] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:39:34] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:40:39] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
Traceback (most recent call last):
  File "webservice.py", line 194, in <module>
    app.run(host='0.0.0.0')
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 772, in run
    run_simple(host, port, self, **options)
  File "/home/steve/.local/lib/python2.7/site-packages/werkzeug/serving.py", line 675, in run_simple
    s.bind((hostname, port))
  File "/usr/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Exception in thread Thread-1 (most likely raised during interpreter shutdown):
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
  File "/usr/lib/python2.7/threading.py", line 763, in run
  File "/home/steve/.local/lib/python2.7/site-packages/werkzeug/serving.py", line 657, in inner
  File "/home/steve/.local/lib/python2.7/site-packages/werkzeug/serving.py", line 497, in serve_forever
  File "/usr/lib/python2.7/SocketServer.py", line 241, in serve_forever
  File "/usr/lib/python2.7/threading.py", line 586, in set
  File "/usr/lib/python2.7/threading.py", line 407, in notifyAll
<type 'exceptions.TypeError'>: 'NoneType' object is not callable
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:42:06] "GET /version HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:42:06] "GET /favicon.ico HTTP/1.1" 404 -
24.55.154.193 - - [18/Mar/2016 02:42:11] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:42:53] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
Exception in thread Thread-1 (most likely raised during interpreter shutdown):
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
  File "/usr/lib/python2.7/threading.py", line 763, in run
  File "/home/steve/.local/lib/python2.7/site-packages/werkzeug/serving.py", line 657, in inner
  File "/home/steve/.local/lib/python2.7/site-packages/werkzeug/serving.py", line 497, in serve_forever
  File "/usr/lib/python2.7/SocketServer.py", line 241, in serve_forever
  File "/usr/lib/python2.7/threading.py", line 586, in set
  File "/usr/lib/python2.7/threading.py", line 407, in notifyAll
<type 'exceptions.TypeError'>: 'NoneType' object is not callable
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
  File "webservice.py", line 43
    updateString = 'UPDATE ProjectTable SET MemberList="' +
                                                          ^
SyntaxError: invalid syntax
  File "webservice.py", line 43
    updateString = 'UPDATE ProjectTable SET MemberList="' +
                                                          ^
SyntaxError: invalid syntax
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:46:17] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:46:18] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:46:18] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:46:43] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:47:49] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/steve/.local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py", line 47, in acceptRequest
    handler.updateRow("ProjectTable", updateString)
  File "/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/SQLiteHandler.py", line 55, in updateRow
    cursor.execute('UPDATE ' + tableName + ' SET ' + updatestring)
OperationalError: near "UPDATE": syntax error
24.55.154.193 - - [18/Mar/2016 02:47:49] "GET /accept?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:47:49] "GET /accept?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:47:49] "GET /accept?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:47:50] "GET /accept?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:47:50] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:47:50] "GET /accept?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:48:11] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:48:13] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:48:13] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:48:13] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:48:14] "GET /accept?projectid=41&addeduser=test HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:49:21] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:49:27] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:49:27] "GET /login?user=steven@test.com&passwd=83116101118101 HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:49:27] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:49:27] "GET /select?search=steven@test.com&record=email&table=UserTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:49:29] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:49:29] "GET /select?search=50&record=ProjectId&table=ProjectTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:49:31] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:49:31] "GET /getall HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:10] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:19] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:19] "GET /login?user=steven@test.com&passwd=83116101118101 HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:19] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:19] "GET /select?search=steven@test.com&record=email&table=UserTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:22] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:22] "GET /select?search=50&record=ProjectId&table=ProjectTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:24] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:24] "GET /getall HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:45] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:45] "GET /select?search=50&record=ProjectId&table=ProjectTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:50:59] "GET /accept?projectid=50&addeduser=tyrsniper94@hotmail.com HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:52:09] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:52:20] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:52:20] "GET /login?user=steven@test.com&passwd=83116101118101 HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:52:20] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:52:20] "GET /select?search=steven@test.com&record=email&table=UserTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:52:22] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:52:22] "GET /select?search=50&record=ProjectId&table=ProjectTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:53:44] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:53:44] "GET /login?user=steven@test.com&passwd=83116101118101 HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:53:44] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:53:44] "GET /select?search=steven@test.com&record=email&table=UserTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:53:45] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:53:45] "GET /select?search=50&record=ProjectId&table=ProjectTable HTTP/1.1" 200 -
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
 * Detected change in '/home/steve/ProjectManagement/FlaskWebService/flask/app/bin/webservice.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 165-611-794
24.55.154.193 - - [18/Mar/2016 02:57:08] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:10] "GET /createaccount?user=abc@abc.com&passwd=116101115116115 HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:10] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:10] "GET /select?search=abc@abc.com&record=email&table=UserTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:19] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:19] "GET /insert?table=ProjectTable&valuestring="abc@abc.com","","","TestingProject","",""&columnstring=leaderlist,memberlist,tasklist,projectname,projectdescription,projectprogress HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:19] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:19] "GET /select?search=TestingProject&record=projectname&table=ProjectTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:19] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:19] "GET /update?table=UserTable&updatestring=projectlist="51"_WHERE_UserID="24" HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:19] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:19] "GET /select?search=24&record=userid&table=usertable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:25] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:25] "GET /getall HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:42] "GET /version HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:57:55] "GET /version HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:58:04] "GET /accept?projectid=51&addeduser=tyrsniper94@hotmail.com HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:58:17] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:58:17] "GET /login?user=abc@abc.com&passwd=116101115116115 HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:58:17] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:58:18] "GET /select?search=abc@abc.com&record=email&table=UserTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:58:19] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:58:19] "GET /select?search=51&record=ProjectId&table=ProjectTable HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:58:35] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:58:35] "GET /getall HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:59:06] "GET /accept?projectid=51&addeduser=sgant869@live.kutztown.edu HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:59:13] "GET / HTTP/1.1" 200 -
24.55.154.193 - - [18/Mar/2016 02:59:13] "GET /select?search=51&record=ProjectId&table=ProjectTable HTTP/1.1" 200 -
